# **HYPOTHESIS TESTING**

Data Provided:

•	The theoretical weekly operating cost model: W = $1,000 + $5X

•	Sample of 25 restaurants with a mean weekly cost of Rs. 3,050

•	Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units.


# **1) State the hypothesis statement**

Hypothesis Testing Mechanism
1) Null Hypothesis (H0) : mean weekly operating cost = theoretical mean weekly operating cost.

H0  u= $ 1000 + $ 5 * X

where X is mean number of units produced, given as 600 units

H0: u= $ 1000 + $ 5 * 600

     = $ 1000 + $ 3000 = $ 4000

2) Alternative Hypothesis (H1) : (Opposite of H0) mean weekly operating cost != theoretical mean weekly operating cost.

 H1 : u!= $ 4000

# **2) Calculate the Test Staistics**

In [1]:
# Given :
#	ˉxˉ = sample mean weekly cost (Rs. 3,050)
#	μ = theoretical mean weekly cost according to the cost model (W = $1,000 + $5X for X = 600 units)
#	σ = 5*25 units
#	n = sample size (25 restaurants)

import numpy as np
from scipy import stats
import math

sample_mean=3050
population_mean=4000
population_std=5*25
sample_size=25

z_value=(sample_mean-population_mean)/(population_std/math.sqrt(sample_size))
print(f"Z-value :{z_value}")

p_value=2*(1-stats.norm.cdf(abs(z_value)))
print(f"P-value :{p_value}")

alpha=0.05

if p_value< alpha:
  print("Reject Null Hypothesis")
  print("Mean weekly operating cost != theoretical mean weekly operating cost")
else:
  print("Accept Null Hypothesis")
  print("Mean weekly operating cost = theoretical mean weekly operating cost")


Z-value :-38.0
P-value :0.0
Reject Null Hypothesis
Mean weekly operating cost != theoretical mean weekly operating cost


# **3) Determine the critical value**

In [2]:
# alpha=0.05
alpha=0.05

from scipy.stats import norm

# Critical Z-value for lower tail
critical_lower = norm.ppf(alpha / 2)

# Critical Z-value for upper tail
critical_upper = norm.ppf(1 - alpha / 2)

print(f"For a two-tailed test with alpha = {alpha}:")
print(f"The critical Z-values are approximately {critical_lower:.4f} and {critical_upper:.4f}")


For a two-tailed test with alpha = 0.05:
The critical Z-values are approximately -1.9600 and 1.9600


# **4) Make a decision**

In [3]:
# Reject Null Hupothesis (H0) if test statistics falls outside the range of critical values.
# For a two tailed test, reject H0 if Z-statistics < critical_lower or z-statistics > critical_upper.
if z_value < critical_lower or z_value > critical_upper:
  print("Decision is Reject Null Hypothesis (H0)")
  print("Z-Statistics falls in rejection region")
else:
  print("Decision is Accept Null Hypothesis (H1)")
  print("Z-Statistics falls outside rejection region")

Decision is Reject Null Hypothesis (H0)
Z-Statistics falls in rejection region


# **5) Conclusion**

Based on the hypothesis test:

1) The calculated Z-statistic (-38.0) falls in the rejection region (outside the critical values of approximately -1.9600 and 1.9600).
2) The p-value (0.0) is less than the alpha = 0.05.
3) Therefore, we reject the null hypothesis (H0).

There is sufficient evidence to conclude that the mean weekly operating cost of the restaurants is significantly different from the theoretical mean weekly operating cost of $4,000.

While the test indicates a significant difference, the calculated sample mean ( 3,050)is actually lower than the theoretical mean( 4,000). This means that based on this sample, there is strong evidence that the weekly operating costs are lower than the model suggests, rather than higher.